coordinator/src/main/java/org/apache/uniffle/coordinator/CoordinatorServer.java

```diff
@@ -201,6 +202,9 @@ private void registerRESTAPI() throws Exception {
     jettyServer.addServlet(
         new CancelDecommissionServlet(this),
         "/api/server/cancelDecommission");
+    jettyServer.addServlet(
+        new RefreshCheckerServlet(this),
+        "/api/server/admin/refreshChecker");
```

Comment:
I think /api/admin/refreshChecker is better.  /api/server is used for some operations on the shuffle servers.
And /api/server will be changed in the future.
https://github.com/apache/incubator-uniffle/pull/684#discussion_r1125665019